pkg/auth/authenticator/bearertoken/bearertoken.go

```diff
@@ -45,3 +46,17 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (user.Info, bool,
 	token := parts[1]
 	return a.auth.AuthenticateToken(token)
 }
+
+// GetOpenAPISecurityDefinition returns a Bearer authentication SecurityDefinition.
+func (a *Authenticator) GetOpenAPISecurityDefinition() (spec.SecurityDefinitions, error) {
+	ret := spec.SecurityDefinitions{}
+	ret["Bearer"] = &spec.SecurityScheme{
+		SecuritySchemeProps: spec.SecuritySchemeProps{
+			Type:        "apiKey",
```

Comment:
Yes, because we pass a "key" with name "Authorization" in header. They key is "Bearer $TOKEN"